src/main/java/io/vertx/kafka/client/common/KafkaClientOptions.java

```diff
@@ -29,13 +33,36 @@
 public class KafkaClientOptions {
 
   private Map<String, Object> config;
+  // Note, tracing policy is not defined by default, as the default value is contextual to usage as consumer or producer
+  private TracingPolicy tracingPolicy;
```

Comment:
I think in both case default TracingPolicy should be PROPAGATE, because we don't want to create a trace by default when no trace exists with Kafka

Reply:
ok, that makes sense.
btw, from your previous commit, the json constructor in options isn't calling the converter, I'm going to add that

Reply:
sure, that's a miss